src/framework/posix.h

```diff
@@ -14,7 +14,7 @@
 
 ! glibc defines sigsetjmp as __sigsetjmp via macro readable from <setjmp.h>.
 #ifndef SIGSETJMP_NAME
-#define SIGSETJMP_NAME "__sigsetjmp"
+#define SIGSETJMP_NAME "sigsetjmp"
```

Comment:
Note to self. Need to check if this works on Linux. On macOS, the two leading underscores need to be removed.